server/src/com/scraniel/auctioneero/AuctionResponse.java

```diff
@@ -1,14 +1,18 @@
-package auctioneero;
+package com.scraniel.auctioneero;
+
+import java.util.UUID;
 
 public class AuctionResponse 
 {
 	private boolean isActionSuccess;
```

Comment:
try having a different prefix/naming convention for private member variables (like mIsActionSuccess or _isActionSuccess).